chia/wallet/wallet_blockchain.py

```diff
@@ -57,30 +54,20 @@ async def create(
         self._latest_timestamp = uint64(0)
         self._height_to_hash = {}
         self._block_records = {}
-        if self.synced_weight_proof is not None:
-            await self.new_weight_proof(self.synced_weight_proof)
```

Comment:
we need to initialize the blockchain when starting up, to set the block records and peak, right?

Reply:
yes, moved this to wallet_node so we can organise the code in a better way and not have references to the weight_proof_handler in multiple places, i think this closer to what we do in the full node.